src/main/scala/edu/cmu/cs/obsidian/codegen/CodeGenerator.scala

```diff
@@ -0,0 +1,11 @@
+package edu.cmu.cs.obsidian.codegen
+
+import java.nio.file.Path
+
+import edu.cmu.cs.obsidian.CompilerOptions
+import edu.cmu.cs.obsidian.parser.SymbolTable
+
+trait CodeGenerator{
```

Comment:
Please add a space before the curly brace.

Reply:
Fixed